### PR TITLE
Multiple bug fix

### PR DESCRIPTION
**Pull request by SPK137**
- add image fallback and actual image for elderlyCard and emerInfoScreen
- fallback for invalid geocoding
- loading overlay for sign up flow
- detect used phone number before continuing to OTP verification for sign up flow
- remove navigation header which should not exist for sign up flow
- add function for re-requesting OTP in sign up flow
- add error handling when OTP and phone verifying endpoint returns error
- add accurate error message when uploaded image exceeds the image size limit

### CODE REVIEW

File: src/internationalization/translations/en.json [hunk 1]
```diff
@@ -328,7 +335,8 @@
     "phone": "Phone Number",
     "openInMaps": "Open in Maps ",
     "callSender": "Call {{name}}",
-    "emergencyLocation": "Emergency Location"
+    "emergencyLocation": "Emergency Location",
+    "addressFallback": "Elderly's device did not allow location tracking. Please contaact directly."
```
Comment (Panoj-k) [inline src/internationalization/translations/en.json]: Typo here

File: src/internationalization/translations/th.json [hunk 2]
```diff
@@ -284,6 +285,12 @@
     "primaryButton": "ตกลง",
     "secondaryButton": ""
   },
+  "uploadImageLimitError": {
+    "header": "Error",
+    "body": "รูปประจำตัวของคุณมีขนาดเกินหนด กรุณาเลือกรูปภาพอื่น",
```
Comment (Panoj-k) [inline src/internationalization/translations/th.json]: Typo here

File: src/internationalization/translations/en.json (hunk 1 above)
Comment (SPK137) [reply src/internationalization/translations/en.json]: FIxed. Thanks

File: src/internationalization/translations/th.json (hunk 2 above)
Comment (SPK137) [reply src/internationalization/translations/th.json]: FIxed. Thanks

**Review by pointmekin — APPROVED** [review]
LGTM

**Pull request merged**